rcongui/src/components/PlayerView/playerList.js

```diff
@@ -192,7 +192,7 @@ const PlayerItem = ({
               className={classes.marginRight}
               target="_blank"
               color="inherit"
-              href={`https://steamcommunity.com/profiles/${steamID64}`}
```

Comment:
Was this one actually a problem?

Now that I think of it, why don't we put https hardcoded for both. There's no reason to downgrade is there?

Reply:
No it wasn't causing an issue as it doesn't fetch anything but yeah :)

Reply:
Can you hardcode https then? And let merge it :)

Reply:
Oh ok, yeah ill push it tomorrow !